core/src/main/java/org/eclipse/hono/config/ClientConfigProperties.java

```diff
@@ -40,6 +40,11 @@
      * when no frames have been received from the remote peer.
      */
     public static final int DEFAULT_IDLE_TIMEOUT = 16000; //ms
+    /**
+     * The default amount of time (milliseconds) after which a a sender link will be closed when no messages have been
+     * sent. The value <code>0</code> disables automatic link timeout.
+     */
+    public static final long DEFAULT_INACTIVE_LINK_TIMEOUT = 0; // ms
```

Comment:
FMPOV you usually use `-1` for such a purpose as `0` would theoretically be a usable value. WDYT?

Reply:
For me the 0 here has the semantic of: a limit (timeout) of 0 is no limit and means unlimited. But I see now that you can read it as "it instantly times out". This would not makes sense for this use case (and the vert.X timer which consumes this value currently does not accept a delay of 0). So, I'm OK with changing this to `-1` for disabling the timeout.